### PR TITLE
VCS integration docs for Terraform Enterprise

### PR DESCRIPTION
**Pull request by clstokes**
/cc @KFishner @bensojona @mootpt

### CODE REVIEW

File: terraform/vcs/git.html.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: "Git Integration"
+---
+
+# Git Integation
+
+Git repositories can be integrated with Atlas by using
+[`terraform push`](https://www.terraform.io/docs/commands/push.html) to import
+Terraform configuration when changes are committed. When Terraform
+configuration is imported using `terraform push` a plan is automatically queued
+in Atlas.
```
Comment (KFishner) [inline terraform/vcs/git.html.md]: I think we should call out in this paragraph that this is Git integration for repos _not_ hosted in GitHub. For GitHub users, there is a [native integration](link).

File: terraform/vcs/github.html.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+proposed changes.
+
+## Setup
+
+Atlas environments are linked to individual GitHub repositories. However, a
+single GitHub repository can be linked to multiple Atlas environments allowing
+a single set of Terraform configuration to be used across multiple environments.
+
+Atlas environments can be linked when they're initially created using the
+**New Environment** process. Existing environments can be linked by setting
```
Comment (KFishner) [inline terraform/vcs/github.html.md]: link to new environments

File: terraform/vcs/git.html.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Terraform configuration can be manually imported by running `terraform push`
+like below:
+
+```
+$ terraform push -name=$ATLAS_USERNAME/ENV_NAME
+```
+
+A better option than having to manually run `terraform push` is to run it
+using a git commit hook. A client-side `pre-push` hook is suitable and will
```
Comment (KFishner) [inline terraform/vcs/git.html.md]: client-side `pre-push` workflow makes sense. 

Thoughts on a CI integration? Would that make sense or should we just suggest the client-side workflow?

File: terraform/vcs/git.html.md (hunk 1 above)
Comment (clstokes) [inline terraform/vcs/git.html.md]: Done.

**Pull request merged**